README.md

```diff
@@ -111,6 +111,9 @@ pipenv shell
 
 # Install all deps from Pipfile.lock
 pipenv sync --dev
+
+# Install all deps from Pipfile
+pipenv install --dev
```

Comment:
This description is not correct. And overall, we shouldn't use the `pipenv install` command.
It updates all requirements in `Pipfile.lock` and only then installs.
As far as we'd like more control on that layer I would highly recommend staying with `pipenv sync --dev`.

`pipenv install` should be used with package and version we are going to install